datasets/huggingface.py

```diff
@@ -89,5 +98,7 @@ def hash(cls, kwargs):
             "name": kwargs["name"],
             "data_files": kwargs["data_files"],
             "revision": kwargs["revision"],
+            "split": kwargs["split"],
+            "token": kwargs["token"],
```

Comment:
Ah I realize, we do custom hashing. The sis hash exclude is ignored then. 

And this here is wrong. It will change the hash.

Reply:
No wait. It is called via `super().hash(d)` or? I think it still use the `__sis_hash_exclude__`.

Reply:
Oh right. Ok then.